source/PluginDev/Assets/GooglePlayGames/BasicApi/DummyClient.cs

```diff
@@ -492,6 +492,22 @@ private static void LogUsage()
         {
             Logger.d("Received method call on DummyClient - using stub implementation.");
         }
+
+        /// <summary>
+        /// Shows the Player Games Profile UI for the given user identifier.
+        /// </summary>
+        /// <param name="userId">Player Id. This cannot be null.</param>
```

Comment:
Player Id - check how we call it in other places, please. Probably 'User identifier'.

Reply:
same in other places